src/app/admin-panel/new-year/statistics/statistics.component.html

```diff
@@ -2,56 +2,53 @@
-              <th mat-header-cell *matHeaderCellDef class="center"> Усього учнів </th>
-              <td mat-cell *matCellDef="let element"> {{element.total}}</td>
-            </ng-container>
-  
-            <ng-container matColumnDef="trend">
-                <th mat-header-cell *matHeaderCellDef class="center"> Тенденція </th>
-                <td mat-cell *matCellDef="let element"> 
-                    <span *ngIf="element.trend">
-                      <ng-container [ngSwitch]="true">
-                        <ng-container *ngSwitchCase="element.trend > 0">&#8599;</ng-container>
-                        <ng-container *ngSwitchCase="element.trend < 0">&#8600;</ng-container>
-                        <ng-container *ngSwitchDefault>-</ng-container>
-                      </ng-container>
-                    </span>
-                </td>
-              </ng-container>
-  
-            <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
-            <tr mat-row *matRowDef="let row; columns: displayedColumns;"></tr>
-          </table>
+          <ng-container matColumnDef="year">
+            <th mat-header-cell *matHeaderCellDef class="center"> Рік </th>
+            <td mat-cell *matCellDef="let element"> {{element.year}} </td>
+          </ng-container>
+          <ng-container matColumnDef="total">
+            <th mat-header-cell *matHeaderCellDef class="center"> Усього учнів </th>
+            <td mat-cell *matCellDef="let element"> {{element.total}}</td>
+          </ng-container>
+          <ng-container matColumnDef="trend">
+            <th mat-header-cell *matHeaderCellDef class="center"> Тенденція </th>
+            <td mat-cell *matCellDef="let element"> 
+              <span *ngIf="element.trend">
+                <ng-container [ngSwitch]="true">
+                  <ng-container *ngSwitchCase="element.trend > 0">&#8599;</ng-container>
+                  <ng-container *ngSwitchCase="element.trend < 0">&#8600;</ng-container>
+                  <ng-container *ngSwitchDefault>-</ng-container>
+                </ng-container>
+              </span>
+            </td>
+          </ng-container>
+          <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+          <tr mat-row *matRowDef="let row; columns: displayedColumns;"></tr>
+        </table>
       </ng-template> 
     </mat-tab>
   </mat-tab-group>
   <i class="material-icons export-pdf" (click)="downloadPDF()" matTooltip = 'Завантажити'>save_alt</i>
-  </mat-dialog-content>
-  <div class ="overlay-tip" *ngIf="showPdfTip">
-    <div class="tip-content">
-        <div>
-          <img class="pdf-icon" src="../../../../assets/file-pdf-regular.svg" alt="pdf">
-        </div>
-        <div>Формується PDF</div>
-    </div>
+</mat-dialog-content>
+<div class ="overlay-tip" *ngIf="showPdfTip">
+  <div class="tip-content">
+      <div>
+        <img class="pdf-icon" src="../../../../assets/file-pdf-regular.svg" alt="pdf">
```

Comment:
FYI: if image is not too big it could be converted to base64 to avoid long pathes